CONTRIBUTING.md

```diff
@@ -51,13 +51,13 @@ run `git submodule update --init --recursive`.
 
 Here are some contribution tips to help you get started:
 
--   Always make sure that you're up-to-date with **Lawnchair** by setting your base branch to `14-dev`.
+-   Always make sure that you're up-to-date with **Lawnchair** by setting your base branch to `15-dev`.
 -   Make sure your code is logical and well-formatted. If using Kotlin, see [“Coding conventions” in the Kotlin documentation](https://kotlinlang.org/docs/coding-conventions.html);
--   [The `lawnchair` package](https://github.com/LawnchairLauncher/lawnchair/tree/14-dev/lawnchair) houses Lawnchair’s own code, whereas [the `src` package](https://github.com/LawnchairLauncher/lawnchair/tree/14-dev/src) includes a clone of the Launcher3 codebase with modifications. Generally, place new files in the former, keeping changes to the latter to a minimum.
+-   [The `lawnchair` package](https://github.com/LawnchairLauncher/lawnchair/tree/15-dev/lawnchair) houses Lawnchair’s own code, whereas [the `src` package](https://github.com/LawnchairLauncher/lawnchair/tree/15-dev/src) includes a clone of the Launcher3 codebase with modifications. Generally, place new files in the former, keeping changes to the latter to a minimum.
 
 #### Additional documentation
 
--   The Lawnchair Wiki ([at Github](https://github.com/LawnchairLauncher/lawnchair/wiki))
+-   The Lawnchair Wiki ([GitHub Wiki](https://github.com/LawnchairLauncher/lawnchair/wiki))
```

Comment:
```suggestion
-   The Lawnchair ([GitHub Wiki](https://github.com/LawnchairLauncher/lawnchair/wiki))
```

Reply:
I think we can leave this text as-is instead.

However, a better approach for consistency is to just directly link the text:

```
* Lawnchair GitHub Wiki
* Lawnchair Visual Guidelines
...
```

Reply:
👋 Hello! (this message was almost 8 hours ago, I just never hit send)

@Goooler The change is intended to be consistent, but looks redundant. I believe @SuperDragonXD suggestion works well. What do you think?

Reply:
Looks good.